modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -28,29 +26,31 @@ The `high-limit` format allows more nodes and relationships to be stored than `s
 
 You can either set the store format when creating a new database or change the store format of an existing database.
 
-The default store format of new databases is `aligned` and is controlled by the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration setting in the _neo4j.conf_ file.
-
 === Creating new databases
 
-There are several ways to create a new database in a specific store format:
+The default store format for all new databases is `aligned`.
```

Comment:
I still don't think this adds any value. But if you really want it, we can at least make a note that this may change in future versions